lib/travis/api/app/endpoint/authorization.rb

```diff
@@ -288,9 +287,10 @@ def fetch
               if user
                 rename_repos_owner(user.login, info['login'])
                 user.update_attributes info
+                Travis.run_service(:sync_user, user) if user.previous_changes[:github_oauth_token]
               else
                 self.user = ::User.create! info
```

Comment:
ah! ok this makes sense - the line I commented on you want to force it to use the writer for @user by using self, but if that's the case, you shouldn't need it for the user reader method in the line you added, as there would be no local variable to reference and it would use the method.